src/lib/application-events.ts

```diff
@@ -0,0 +1,3 @@
+export const ApplicationEvents = {
```

Comment:
I'm going to end up putting a `Application.EVENT_BOOTSTRAP_END` on the `Application` class as well for discoverability.